generators/sync-file-system.js

```diff
@@ -178,5 +187,16 @@ class Benchmark {
                 }
             }
         }
+
+        for (let _ of fs.forEachFileRecursively()) {
+            this.totalFileCount++;
+        }
+    }
+
+    validate(iterations) {
+        if (this.totalFileCount != this.EXPECTED_FILE_COUNT * iterations)
+            throw new Error(`Invalid total file count ${this.totalFileCount}`); 
```

Comment:
Nit: you could include the expected count too.

Reply:
done.